share/functions/help.fish

```diff
@@ -78,6 +78,11 @@ function help --description 'Show help for the fish shell'
 		return 1
 	end
 
+	# In Cygwin, start the user-specified browser using cygstart
+	if begin; type -q cygstart; and test $fish_browser != "cygstart"; end
+		set fish_browser cygstart \"$fish_browser\"
+	end
+
```

Comment:
I always find this style very difficult to read and maintain because the line is too long.

```fish
if type -q cygstart
    if test "$fisher_browser" != "cygstart"
        set fish_browser cygstart \"$fish_browser\"
    end
end
```